### PR TITLE
BrowserView gets disconnected randomly #302

### PR DESCRIPTION
**Pull request by rd1992**

### CODE REVIEW

**Comment by damithc** [comment]
Use the following format for pull request titles:
Issue title #issue_number

File: src/browserview/BrowserComponent.java [hunk 1]
```diff
@@ -187,11 +199,17 @@ public void newMilestone() {
...
-				driver.get(GitHubURL.getPathForNewIssue());
-			}
-		});
+		if (isBrowserActive()) {
+			runBrowserOperation(() -> {
+				if (!driver.getCurrentUrl().equals(GitHubURL.getPathForNewIssue())) {
+					driver.get(GitHubURL.getPathForNewIssue());
+				}
+			});
+		}
```
Comment (damithc) [inline src/browserview/BrowserComponent.java]: Use this format `} else {`

File: chromedriver-log.log [hunk 2]
```diff
@@ -0,0 +1,197 @@
+[1.052][INFO]: COMMAND InitSession {
```
Comment (damithc) [inline chromedriver-log.log]: Is this file necessary?

**Comment by damithc** [comment]
Commented on some style issues. Didn't review the code.

File: src/browserview/BrowserComponent.java [hunk 3]
```diff
@@ -161,11 +161,16 @@ private void hidePageElements() {
 	 */
 	public void newLabel() {
 		logger.info("Navigating to New Label page");
-		runBrowserOperation(() -> {
-			if (!driver.getCurrentUrl().equals(GitHubURL.getPathForNewLabel())) {
-				driver.get(GitHubURL.getPathForNewLabel());
-			}
-		});
+		if (isBrowserActive()) {
```
Comment (dariusf) [inline src/browserview/BrowserComponent.java]: We could move this check into `runBrowserOperation`. That way we'd only have to update it in one place every time. Plus we're guaranteed that uses of `driver.getCurrentUrl()` won't be null/empty in the body of tasks, etc.

File: src/browserview/BrowserComponent.java [hunk 4]
```diff
@@ -235,6 +265,34 @@ public void showIssue(int id) {
...
+				default:
+					break;
+				}
+				return false;
+			}
+	}
+	
+	private void resetBrowser(){
+		logger.info("Chrome not responding.");
+		quit();
```
Comment (dariusf) [inline src/browserview/BrowserComponent.java]: `quit` was meant to be called only once, on application exit. Since `driver.quit()` kills the process, I guess it's no longer needed.
